src/components/logo/logo.js

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+
+export default class Logo extends React.Component {
+  constructor() {
```

Comment:
Do you get an error if you take out the `constructor` here? This seems unnecessary. Also, fwiw, this could just be a functional component instead of a class.